index.html

```diff
@@ -31,61 +28,68 @@
-          of Product & Engineering.
+          I have not held a positions as a Software Engineer. I just had an Insurance Agent position but things can
+          change tomorrow.
+          Just like things change for this catch!
         </p>
       </div>
     </div>
     <div class="divider">Projects</div>
     <div class="section projects">
       <div class="project">
         <div class="explanation">
-          <p>Animated floating Hal Jordan, the Green Lantern.</p>
-          <p>Image is an SVG created in Sketch.</p>
-          <p>Animation is achieved with CSS.</p>
-          <a href="https://github.com/jlcarmic/green-lantern" target="_blank">
-            <button class="button">Green Lantern</button>
+          <p>I'm Happy To Let You Know That My Inner Child Is Still Ageless</p>
+          <p></p>
+          <p></p>
+          <a href="https://www.goodreads.com/quotes/tag/inner-child" target="_blank">
+            <button class="button">Inner Child In You Quotes</button>
           </a>
         </div>
         <div class="screenshot">
-          <img src="images/greenlantern.png" />
+          <img src="./images/superboy.jpg" />
         </div>
       </div>
 
       <div class="project">
         <div class="screenshot">
-          <img src="./images/producthuntalexa.png" />
+          <img src="./images/The-world-is-yours.jpg" />
         </div>
         <div class="explanation">
-          <p>An Alexa Skill for hearing Today's Featured products.</p>
+          <p>An Alexa Skill that helps you find the continent of any country.</p>
           <p>Built using NodeJS and AWS Lambda.</p>
-          <p>Utilizes ProductHunt's API.</p>
+          <p></p>
           </p>
-          <a href="https://github.com/jlcarmic/producthunt_alexa_skill" target="_blank">
-            <button class="button">Today's Featured</button>
+          <a href="https://github.com/pedroderlrosario1/world-countries/pull/1" target="_blank">
+            <button class="button">World Countries</button>
           </a>
         </div>
       </div>
 
       <div class="project">
         <div class="explanation">
-          <p>An NPM package for accessing sports<br />stats via the Stattleship API.</p>
-          <p>Built in NodeJS and published to NPM<br />as an open source package.</p>
-          <a href="https://github.com/jlcarmic/node-stattleship" target="_blank">
-            <button class="button">Node Stattleship</button>
+          <p>I'm also a Dancer<br />A model</p>
+          <p>And a Kool Cat<br /></p>
+          <a href="https://www.weddingwire.com/biz/promises-to-keep-derry/627357906aeb72c0.html" target="_blank">
+            <button class="button">Find me in Promises to keep in Derry NH</button>
           </a>
         </div>
         <div class="screenshot">
-          <img class="stattleship" src="./images/stattleship.svg" />
+          <img class="Kool Cat" src="./images/CoolCat.jpg" />
```

Comment:
CSS classes are for applying CSS styles. They also can't have spaces in them, this is trying to apply two sets of styles to this image though neither class exists in your CSS.